docs/dev/table_api.md

```diff
@@ -1022,69 +1022,70 @@ Temporal intervals can be represented as number of months (`Types.INTERVAL_MONTH
 
 ### Windows
 
-The Table API is a declarative API to define queries on batch and streaming tables. Projection, selection, and union operations can be applied both on streaming and batch tables without additional semantics. Aggregations on (possibly) infinite streaming tables, however, can only be computed on finite groups of records. Group-window aggregates group rows into finite groups based on time or row-count intervals and evaluate aggregation functions once per group. For batch tables, group-windows are a convenient shortcut to group records by time intervals.
+The Table API is a declarative API to define queries on batch and streaming tables. Projection, selection, and union operations can be applied both on streaming and batch tables without additional semantics. Aggregations on (possibly) infinite streaming tables, however, can only be computed on finite groups of records. Window aggregates group rows into finite groups based on time or row-count intervals and evaluate aggregation functions once per group. For batch tables, windows are a convenient shortcut to group records by time intervals.
 
-Group-windows are defined using the `window(w: GroupWindow)` clause. The following example shows how to define a group-window aggregation on a table.
+Windows are defined using the `window(w: Window)` clause, when define a window it must be assigned the window an alias using `as`. Window aggregates must with additional `groupBy` attribute, it should be noted that the window alias must be included in `groupBy` statement.
```

Comment:
Windows are defined using the `window(w: Window)` clause. When defining the window, it must assign an alias to the window, using `as`.  It should be noted that in window aggregation, the window alias must be referenced in an additional `groupBy` attribute. The following example shows how to define a window aggregation on a table.